sui-execution/latest/sui-adapter/src/static_programmable_transactions/env.rs

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
Is this the compilation environment?

Reply:
It holds all the shared environments used for "compilation" and execution.

Reply:
Adding a comment